src/app/CommandHandler.cpp

```diff
@@ -108,10 +110,16 @@ CHIP_ERROR CommandHandler::ProcessCommandDataElement(CommandDataElement::Parser
 exit:
     if (err != CHIP_NO_ERROR)
     {
-        // The Path is not present when there is an error to be conveyed back. ResponseCommandElement would only have status code,
-        // set the error with CHIP_NO_ERROR, then continue to process rest of commands
-        AddStatusCode(nullptr, GeneralStatusCode::kInvalidArgument, Protocols::SecureChannel::Id,
-                      Protocols::SecureChannel::kProtocolCodeGeneralFailure);
+        chip::app::CommandPathParams returnStatusParam = { endpointId,
+                                                           0, // GroupId
+                                                           clusterId, commandId, (chip::app::CommandPathFlags::kEndpointIdValid) };
+
+        // The Path is the path in the request if there are any error occurred before we dispatch the command to clusters.
+        // Currently, it could be failed to decode Path or failed to find cluster / command on desired endpoint.
+        // Set the error with CHIP_NO_ERROR, then continue to process rest of commands
```

Comment:
I have no idea what this is saying.  What does "Set the error with CHIP_NO_ERROR" mean?

Reply:
Updated this code, and "Always return CHIP_NO_ERROR since we have returned application level error code, and we can proceed to the next command in the same invoke request" above the return statement.